src/java/edu/ucsd/library/dams/triple/TripleStoreUtil.java

```diff
@@ -320,6 +335,49 @@ private static void loadRDF( InputStream in, boolean deleteFirst,
 		{
 			throw new TripleStoreException( "Error processing triples", ex );
 		}
+
+		return null;
+	}
+	public static Set<String> validateModel( Model model, Set<String> validClasses,
```

Comment:
That could be a good idea, since I wouldn't be surprised if we do wind up implementing more validation operations in the short/medium term.  Though I was hoping to keep as much of that as possible in Ruby to make it as easy as possible to migrate.